keps/sig-scheduling/5004-dra-extended-resource/README.md

```diff
@@ -944,7 +944,13 @@ Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
 checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
-Will be considered for beta.
+`kube_pod_resource_limit` and `kube_pod_resource_request`
+(label: `namespace`, `pod`, `node`, `scheduler`, `priority`, **`resource`**, `unit`)
+can be used to determine if the feature is in use by workloads though it doesn't differentiate 
+between extended resources backed by DRA or device plugin.
+
+`resourceclaim_controller_creates_total` (label: `status`, `admin_access`, `source`)
```

Comment:
This needs to be implemented additionally.
`resourceclaim_controller_creates_total` doesn't have `source` label.
We can determine if the resource claim is created by resource claim template, or extended resource by its annotation.
resource claim created by resource claim template has `resource.kubernetes.io/pod-claim-name` and one created by extended resource has `resource.kubernetes.io/extended-resource-claim`.

We can add this label like admin_access.
https://github.com/kubernetes/kubernetes/blob/5c4f81743c2ee73fae71d336304e491b2539b6e3/pkg/controller/resourceclaim/controller.go#L978-L989